tests/util.py

```diff
@@ -2,14 +2,24 @@
 import time
 import binascii
 import os 
-
+import tempfile
+import shutil
 import mock
 
 
 DEFAULT_TIMESTAMP = datetime.datetime(2016, 1, 1)
 DEFAULT_ISO_TIME = time.mktime(DEFAULT_TIMESTAMP.timetuple())
 
 
+def mk_db_and_blob_dir():
+    db_dir = tempfile.mkdtemp()
+    blob_dir = tempfile.mkdtemp()
+    return db_dir, blob_dir
+
+def rm_db_and_blob_dir(db_dir, blob_dir):
+    shutil.rmtree(db_dir, ignore_errors=True)
```

Comment:
there is a race condition that gets triggered on windows , but it could probablly happen on other OS (BlobFile class creates a temporary file which it deletes, rmtree will try to delete that file but it will have already been deleted)  
https://bugs.python.org/issue29699